dali/test/python/test_eager_operators.py

```diff
@@ -87,3 +92,77 @@ def test_arithm_op_context_manager_deep_nested():
 
     assert np.array_equal((tl_1 + tl_2).as_array(), expected_sum)
     eager.arithmetic(False)
+
+
+def test_identical_rng_states():
+    eager_state_1 = eager.rng_state(seed=42)
+    eager_state_2 = eager.rng_state(seed=42)
+
+    out_1_1 = eager_state_1.random.normal(shape=[5, 5], batch_size=8)
+    out_1_2 = eager_state_1.noise.gaussian(out_1_1)
+    out_1_3 = eager_state_1.random.normal(shape=[5, 5], batch_size=8)
+
+    out_2_1 = eager_state_2.random.normal(shape=[5, 5], batch_size=8)
+    out_2_2 = eager_state_2.noise.gaussian(out_2_1)
+    out_2_3 = eager_state_2.random.normal(shape=[5, 5], batch_size=8)
+
+    assert np.allclose(out_1_1.as_tensor(), out_2_1.as_tensor())
+    assert np.allclose(out_1_2.as_tensor(), out_2_2.as_tensor())
+    assert np.allclose(out_1_3.as_tensor(), out_2_3.as_tensor())
+
+
+def test_identical_rng_states_interleaved():
+    eager_state_1 = eager.rng_state(seed=42)
+    eager_state_2 = eager.rng_state(seed=42)
+
+    out_1_1 = eager_state_1.random.normal(shape=[5, 5], batch_size=8)
+    eager_state_1.random.normal(shape=[6, 6], batch_size=8)
+    eager_state_1.noise.gaussian(out_1_1)
+    out_1_2 = eager_state_1.random.normal(shape=[5, 5], batch_size=8)
+
+    out_2_1 = eager_state_2.random.normal(shape=[5, 5], batch_size=8)
+    out_2_2 = eager_state_2.random.normal(shape=[5, 5], batch_size=8)
+
+    assert np.allclose(out_1_1.as_tensor(), out_2_1.as_tensor())
+    assert np.allclose(out_1_2.as_tensor(), out_2_2.as_tensor())
+
+
+def test_objective_eager_resize():
```

Comment:
Please add a comment that this tests the hidden functionality of exposing the Eager Ops classes.